service/instance.py

```diff
@@ -1521,13 +1522,25 @@ def _to_user_driver(core_identity):
         (auth, sess, token) = _token_to_keystone_scoped_project(
             auth_url, auth_token,
             project_name, domain_name)
+        user_driver = UserManager(auth_url=auth_url,
```

Comment:
@kamfonik  It is just covering the bases in case some of the above are not there. I agree that we should have a one-time login and then pass around the `session`. But this requires a deep clean for `rtwo` and I don't think we could finish this before the summit ( have other priorities). So this more like a tempory solution.